docs/subnets/create-a-simple-rust-vm.md

```diff
@@ -4,7 +4,6 @@ This is part of a series of tutorials for building a Virtual Machine (VM):
 
 - [Introduction to VMs](./introduction-to-vm.md)
 - [How to Build a Simple Golang VM](./create-a-vm-timestampvm.md)
-- [How to Build a Complex Golang VM](./create-a-vm-blobvm.md)
```

Comment:
We should keep this doc for now.

Reply:
@yulin-dong Should I add a warning to this page that the repository is archived?

Reply:
Hmm, for now, let's leave as is and readers will find out that the project has been archived when they click on the link. The principals still apply to build a VM.